Feature/added value design

bmlancien opened the pull request.

bmlancien commented:
@henhuy Is it planned to use Ion RangeSlider? If not I will try to change the color of the slider in "Rotordurchmesser" and "Anlagenleistung" with CSS/JS, because right now it is the default blue color of the browser, but green would fit better:

![chrome-capture-2025-3-24](https://github.com/user-attachments/assets/383b856e-498b-4c34-b961-321c2b11f6a0)

henhuy commented:
Hey @bmlancien,
It's up to us which sliders we want to use.
In my memory, we used Ionrangesliders as it comes with many options and customization (i.e. slider marks).
On the other hand, it is not longer supported (last commit 2019) and needs jQuery, which I want to get rid of if possible.
So if you have an alternative by hand, I would be eager to test it!

bmlancien commented:
> So if you have an alternative by hand, I would be eager to test it!

@henhuy I haven't found something yet that would be worth to replace it. I've only seen older libraries too. But I can have a deeper look, maybe i find something else. I've updated the color of the sliders to green just with CSS. It's an okay solution, I would say. As long as we don't need more customized sliders.

> needs jQuery, which I want to get rid of if possible.

Talking about that, I added some [jQuery code](https://github.com/stadt-land-energie-projekt/sl-app/pull/181/commits/241bd6ff4b3572cf06e03c7526108d3cca5756dd#diff-6b69ea714eedde3bb2178eb4eb0dfcce65e94856b648bd8d73d5cf1606bc084bR976) for the accordion. I was not able to make it fully functional in JS yet. Do you want me to find a solution in JS or is it fine this way?

henhuy commented:
> Talking about that, I added some [jQuery code](https://github.com/stadt-land-energie-projekt/sl-app/pull/181/commits/241bd6ff4b3572cf06e03c7526108d3cca5756dd#diff-6b69ea714eedde3bb2178eb4eb0dfcce65e94856b648bd8d73d5cf1606bc084bR976) for the accordion. I was not able to make it fully functional in JS yet. Do you want me to find a solution in JS or is it fine this way?

Yes, it's fine. Getting rid of jQuery is a mid term thing. ;)

bmlancien commented:
> Yes, it's fine. Getting rid of jQuery is a mid term thing. ;)

Alright, in that case, this is ready for review :)

nesnoj reviewed (changes requested):
Thx @bmlancien for restyling!
Currently the area fields cannot be edited for any technology:

![image](https://github.com/user-attachments/assets/ce97eb71-90e0-415f-97a4-f7f4abccea67)

bmlancien commented:
> Currently the area fields cannot be edited for any technology:

@nesnoj Thanks for pointing this out! I just fixed this.

nesnoj commented:
Thx!
One last thing: could you please move the left text to the top to make it more consistent to the app layout?

henhuy commented:
> @henhuy Can you help me using the right layout for this page (without sidebar)? I'm not able to make it work properly.

Done. Header needs some design love now. At least in added value results. ;) @bmlancien

bmlancien commented:
Thanks @henhuy for the help! :)

Is it better now @nesnoj ?

nesnoj reviewed (approved):
LGTM now, thx!
I opened another issue for the added value calculator results: #186
